src/components/SaveToPhone.js

```diff
@@ -24,20 +30,16 @@ class SaveToPhone extends React.Component {
   };
 
   handleOnSubmit = () => {
-    let page;
+    let path = "";
     // Needed for gatsby builds that don't have references to window
     if (typeof window !== "undefined") {
-      page = window.location.href;
+      path = window.location.pathname;
     }
-    const message = `Eviction Free NYC!
-
-Follow this link for assistance in your eviction case: ${page}
-    `;
 
     this.setState({ button: "loading" });
 
-    APIClient.sendSMS(this.state.phone, message).then((res) => {
-      if (res.statusCode >= 400) {
+    sendSMS(this.state.phone, path).then((res) => {
+      if (res.status >= 400) {
```

Comment:
Since this is the result of a `fetch` you might want to use the [`ok` property](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok) (i.e. `if (res.ok)`) here instead, it's a bit more readable.  It also will be `false` if the result was a redirect (HTTP status code in the 300-399 range), which it never should be.

Reply:
Noice